### PR TITLE
Fix -247 -Check your email should open gmail in seprate tab

### LINKED ISSUE #247: Adding Email providers
**I'm submitting a ...**  (check one with "x")
- [x] feature request

**Actual Behaviour**

screenshot from 2018-10-04 19-37-26

**Expected Behaviour**

Should open Gmail in new tab.
**Steps to reproduce it**

**Other information:**

**System information:**

```
Your operating system
```
Ubunutu
**Would you like to work on the issue?**
Yes

### PR DESCRIPTION
**Pull request by sk9331657**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] There is a corresponding issue for this pull request.
- [x] My branch is up-to-date with the Upstream `master` branch.

#### Short description of what this resolves:
Adding route for check your email

screenshot from 2018-10-04 19-45-32

Fixes #247  #249

### CODE REVIEW

**Comment by sk9331657** [comment]
@abishekvashok  Can you please merge it.

**Comment by sk9331657** [comment]
@tabesin  So, like earlier js is also not working. So, what action is actually required.

**Comment by sk9331657** [comment]
> @tabesin So, like earlier js is also not working. So, what action is actually required?

**Comment by sk9331657** [comment]
@tabesin  javascript:window.close(); was not working so i thought to add this change.

**Comment by sk9331657** [comment]
@meets2tarun  Can you explain the earlier working on clicking of this button??

**Comment by tabesin** [comment]
It is just an information screen that tells you to check your mail. There is no further interaction with the App after this screen appears.

**Comment by tabesin** [comment]
It would be an option to include this feature depending on whether the person provided a gmail.com or googlemail.com address. Then it would be a custom button like "Check your Gmail".

Given that the app is quite rough in the sense that it is not used many times by users and maybe may never get more than 1000 users you have to consider whether it is worth to add that feature which is a lowest priority luxury "nice to have" and should be capable to get maintained. The only valid acceptable application would be what I just sketched. Check if the mail address is plain gmail, then provide your feature.

File: templates/build.html [hunk 1]
```diff
@@ -1,21 +1,71 @@
...
-<html >
+<html>
+
 <head>
-  <meta charset="UTF-8">
-  <title>Meilix Generator Email Notification</title>
-      <link rel="stylesheet" href="{{ url_for('static',filename='css/email.css') }}">
+	<meta charset="UTF-8">
+	<title>Meilix Generator Email Notification</title>
+	<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
```
Comment (tabesin) [inline templates/build.html]: We'd use bootstrap locally as a dependency.

File: templates/build.html [hunk 2]
```diff
@@ -1,21 +1,71 @@
...
-	<section class="envelope-top"></section>
-	<section class="envelope-left"></section>
-	<section class="envelope-right"></section>
-</main>
-	<a class="tooltip" href="javascript:window.close();">Check your email!</a>
+	<div class="custom-navbar">
+
+		<div class="alert alert-info" style="text-align:center;">
+
+			<a style="text-decoration: none;font-size:2em" href='.'>
```
Comment (tabesin) [inline templates/build.html]: do not use inline css

File: templates/build.html [hunk 3]
```diff
@@ -1,21 +1,71 @@
...
+	<main>
+		<div class="msg">1</div>
+		<section class="envelope-top"></section>
+		<section class="envelope-left"></section>
+		<section class="envelope-right"></section>
+	</main>
+	<div class="container">
+
+
+		<div class="alert alert-info" style="text-align:center">
```
Comment (tabesin) [inline templates/build.html]: avoid inline css where possible

File: templates/build.html [hunk 4]
```diff
@@ -1,21 +1,71 @@
...
...lP6VZGFHx76HuToNk8yhWn0jB0Y4ZtrlTfXrcyfItvWuFzS2se64pgBoxtM7mdzOTkJRTct4qD5Y+0dvN4V00zIpfRf+hRrPzjVoZPPKZL5w9Bs1mEC2Dfo/OIz+SwKWH3tMRrXdjqzbSTdkGuNaw/Qac72JQI70uqcQ4nQatdmGsN0xfPFO/+2AKEPi20drtA1xpm6HcUa2XIfsUL7yGHxbpSMLYqEzcn0WC3qh5YDgt2pWCGMUNyKRhWOGtrrRJcaVjUyvYKlkdH2kmrFFcGhm5KcixYaS2CVqskVyaV/Xu0/NNPBjm8D7RWea4MTGcz+S6sqGl9s0pyZWDG4jkU99DVU8FSYxGcnJyU6srC+tDKkDaqxrvMu0BbsUpzZWH6A97K4lmasUp3ZWGG8Yi8bcGaVidmle/agIk3JWzdPwo0LXaVXF95mL544q632NZiwViy4pVLpcEp/6hptTVNqmsTpk82lzdswTMJNNmuXMT0WaaVwQLxuZmwpNUXD2aM1stnNkvfaeshMV6ciK1aGduArHIo3ZWPmP4Y/04UbC2aaZZkVx7Wn8CqjG0PMyq59cWPGGxKaO9hgyU7XhyYYdzfaLkh3cWBBZvBqsSVa7AneZX8+srBdJ6qknhlNyN8VjWuNawlYFXkSmDc0qqsvhKYKIcVxovBmgirOletJk5ihXmsKQoKqyheY3avF4NV4xrH98YRWCWu1XsJYlgV9ZV660UIkx+vceY9IRFMuivLEsJkuzZZIpjk+sq/FSqASY1XPljRGHNgMl0CFjdiEl1CFg8mr77EKl4qpcULZeUjJsmF5JAPk5PHraocTEq8dmEBrC3XtT2HCawl07UzKxOx0utrD1YaVna89lGlYSW79mStYaW69sphFlZmfR2gWsFKjNdhrO92UZrrkBymIlaW6whWBDuivjzEdRSLwQ6P1+bnWNOu41QM1jzEVT/7/aosQ0fgOjJYa9je8fr550K5mGcTuXIVwcrAdnXVz96G4Hq3M+8fJq5CVEpUY/vGC8LFPoc/yPz+5LerKFYUsb1c9d9vr/H3FgxszzFXwwVXMTlMYJ2trtTnO87evk6T73lY9kLqe9/D750XygKYvrYkmNRo/nr7vRpvP+9fV98/AbLpzWUyXpS/Cx5jzoIuNSZf/3yPP/fD0xj0PyVcoXEK+DOGAAAAAElFTkSuQmCC">
```
Comment (tabesin) [inline templates/build.html]: avoid inline css

File: templates/build.html [hunk 5]
```diff
@@ -1,21 +1,71 @@
...
...mEXIY/UVc6uHnr0iehmpin8WfncNl4NF3YRgidbeGakdLP+dxY6utu66X57okRs11JOtg0ISlc01XFpXwtR/OsJRXhuPjjMLJ0sGlCroihOqMd8Laj3pFkNe9o++dc5NlSw4RccquqMQllXl13RlbVP7HPxqjaWbWh2f6hXzQMVejt8mmA7ji+tG+xpGmisal4KNYwIVdlkhcThWqVJo1UEZ6DGXqoXAs3TMgnt5/S9efxJV51kqUkrCXThFwHfyZ9iJS3H21F2EJCzhA7O0ktCguVQW1V30JCfv6yPj3lg9KFtjNjIaHQe1reXIeEvoK+k28jIb9MO31hEYrV60TfyLeRUPx6Sp+7EonleXUSaS8hEWKpD/q3DjEVW1+6Go1RQpMn97DYKIq/KQowDnx3d1MNlLVQrSeUVDWj+X63+1zdFqV1pVKLCbnCt16nsv6vnYQOqpCXslFKj0JbSujQaocvx+VnvW0ldHCVA7SdY/n3Lqz0FhnhtsLMfip8I8FaQn3LlulUekDYakIHByU1hn4VQJsJSyp90abal2asJnTCjfIx9sKKX+6ym9DB7qfUbYx+nvTNrsYJzxsO/emJc5z03ad9O+9KaPYke1EYkdN6kWfFyWiwR1UXaFsIU0g/dImzPRz99GuyNfBaQ5hzvuqvPzyN8D4BIRACIRACoUWE5r/pbIrwYUBG2PAvfzwg3H9Yh3QcWnstPO13ovDDykYpO+v/QkIzCmtvNC0jxNtyojYTYlT6+w6vJAzJnWI/2IY8Z9uvWoJ/CWEYx/FYVHzRKB5lml2U/V+cvSeKkod8aYs8PhAqpD3mA4RACIRACIRACIRACIRACIRACIRACIStIezRBv6QulWE0XT4ebzjF8jbQ5ipO+stT16jf1/9xYRM0WL+u0MENcJpBSHTeLH+2oQEeWY5LSJkiierzy05c5rCtI4wU3c0HX4fqZFtyE5CpmQ2+DgFbvgYp82ETNHsvA2F92+39hMyZdsQuWcbagsh03kb2tfdhtpFmCnbhpzK5tlCQqZ0G+pXiYZaS8h0Nc93JWSKJ+cgXuU934IwUzddtsi9CW7fh5DpvGz3B1o8UfHvzQiZ8gMW7JTFqycDAoFAIBAIBAKBQCAQCAQCgUAgEAgEAoFAIBAIBAKBQCAQCAQCgUA26j8BH6YjiT3piQAAAABJRU5ErkJggg==">
```
Comment (tabesin) [inline templates/build.html]: Put images in external files, not in tags

**Comment by sk9331657** [comment]
@tabesin  @abishekvashok  please review.

**Comment by r0mflip** [comment]
IMHO, please don't limit to a specific list of email providers.

**Comment by sk9331657** [comment]
@r0mflip We can add more providers at a later stage.

File: templates/build.html (hunk 1 above)
Comment (tabesin) [inline templates/build.html]: <link rel="stylesheet" type="text/css" href="{{ url_for('static',filename='css/bootstrap.min.css') }}">

Comment (tabesin) [reply templates/build.html]: `<link rel="stylesheet" type="text/css" href="{{ url_for('static',filename='css/bootstrap.min.css') }}>"`

**Pull request merged**